string class_name
float32 confidence
BoundingBox2D bbox_2d